cloudhub-2/modules/ROOT/pages/_partials/create-config.adoc

```diff
@@ -25,9 +25,13 @@ the allowed block size that you can assign for your Anypoint VPC is between `/16
 
 When specifying CIDR blocks, ensure that the CIDR blocks:
 
-* Are from a private IP space
-* Don't overlap with any other CIDR blocks assigned to your other private spaces
-* Don't overlap with any CIDR blocks in use in your corporate network
+* Are from a private IP space.
+* Don't overlap with any other CIDR blocks assigned to your other private spaces.
+* Don't overlap with any CIDR blocks in use in your corporate network.
+
+MuleSoft recommends using subnets from standard RFC 1918 addresses, such as `10.0.0/8`, `172.16.0.0/12`, and `192.168.0.0/16`.
```

Comment:
@sarathecoubian Line 32 should have `10.0.0.0/8` instead of `10.0.0/8`

Reply:
```suggestion
MuleSoft recommends using subnets from standard RFC 1918 addresses, such as `10.0.0.0/8`, `172.16.0.0/12`, and `192.168.0.0/16`.
```